test/onnx/test_pytorch_onnx_onnxruntime_cuda.py

```diff
@@ -27,21 +27,24 @@ def forward(self, x):
 
     @skipIfUnsupportedMinOpsetVersion(9)
     @skipIfNoCuda
+    @disableScriptTest()
     def test_layer_norm_fp16(self):
         class LayerNormModel(torch.nn.Module):
             def __init__(self):
                 super(LayerNormModel, self).__init__()
                 self.layer_norm = torch.nn.LayerNorm([10, 10])
 
+            @autocast()
             def forward(self, x):
                 return self.layer_norm(x)
 
         x = torch.randn(20, 5, 10, 10, requires_grad=True, dtype=torch.float16, device=torch.device("cuda"))
-        self.run_test(LayerNormModel(), x, rtol=1e-3, atol=1e-5)
+        self.run_test(LayerNormModel().cuda(), x, rtol=1e-3, atol=1e-5)
```

Comment:
Do we need .cuda() for FusionModel as well?

Reply:
that one does not have parameters, so omitting does not affect it.